CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 1.2.1
+* Fix in modal selection issue.
+* Fix table content replacement issue.
```

Comment:
@anandraj346 Let's remove this and put in a separate PR specifically pointed to `release-1.3.0` branch.

Reply:
Sure